history/cultures/alteraci.txt

```diff
@@ -0,0 +1,51 @@
+1.1.1 = {
```

Comment:
Night elves have to techs.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/103179894-dd099e00-48a9-11eb-81e2-7e4c8fd04b58.png)

</details>